github-webooks/s3.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_s3_bucket" "webhooks_bucket" {
+  bucket = "moz-github-webhooks"
+  acl = "private"
+}
```

Comment:
One of the few resources in this plan that supports tags! I have a (draft, but no changes suggested yet) tagging standard at https://mana.mozilla.org/wiki/display/DEVSERVICES/Resource+Tagging; some of that feels a lot for an S3 bucket, so I'm happy for feedback.